tests/unit/test_charm.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+#
+# Learn more about testing at: https://juju.is/docs/sdk/testing
+
+import json
+import unittest
+
+from ops.model import ActiveStatus, BlockedStatus
+from ops.testing import Harness
+
+from charm import PrometheusScrapeTargetCharm
+
+
+class TestCharm(unittest.TestCase):
+    def setUp(self):
+        """Flake8 forces me to write meaningless docstrings."""
+        self.harness = Harness(PrometheusScrapeTargetCharm)
+        self.harness.set_model_info(name="lma", uuid="e40bf1a0-91f4-45a5-9f35-eb30fd010e4d")
+        self.addCleanup(self.harness.cleanup)
+        self.harness.begin()
+
+    def test_no_data(self):
```

Comment:
Test what about `no_data` ? This test name could be better worded such as `test_charm_blocks_without_any_targets` or some such thing. Such a name gives a meaningful but cryptic summary of what the test is all about with requiring the developer to read the whole test to understand what is being tested.
Note the code comments and docstrings are usually not visible when tests fail in a test runner.